packages/pn-personafisica-webapp/src/pages/NuovaDelega.page.tsx

```diff
@@ -202,32 +246,43 @@ const NuovaDelega = () => {
                   {({ values, setFieldValue, touched, errors, setFieldTouched }) => (
                     <Form>
                       <FormControl sx={{ width: '100%' }}>
-                        <RadioGroup
-                          aria-labelledby="radio-buttons-group-pf-pg"
-                          defaultValue="pf"
-                          name="selectPersonaFisicaOrPersonaGiuridica"
-                          value={values.selectPersonaFisicaOrPersonaGiuridica.toString()}
-                          onChange={(event) => {
-                            setFieldValue(
-                              'selectPersonaFisicaOrPersonaGiuridica',
-                              event.currentTarget.value
-                            );
-                          }}
-                        >
-                          <Grid
-                            container
-                            sx={{ width: '100%', justifyContent: 'space-between' }}
-                            className={classes.direction}
-                          >
-                            <Grid item xs={isMobile ? 12 : 3}>
+                        <Stack direction={isMobile ? 'column' : 'row'}>
+                          <Stack justifyContent="center">
+                            <RadioGroup
+                              aria-labelledby="radio-buttons-group-pf-pg"
+                              defaultValue={RecipientType.PF}
+                              name="selectPersonaFisicaOrPersonaGiuridica"
+                              value={values.selectPersonaFisicaOrPersonaGiuridica.toString()}
+                              onChange={(event) => {
+                                setFieldValue(
+                                  'selectPersonaFisicaOrPersonaGiuridica',
+                                  event.currentTarget.value
+                                );
+                              }}
+                            >
                               <FormControlLabel
-                                value="pf"
+                                value={RecipientType.PF}
```

Comment:
you should add the `onClick` prop with exactly the same code as for the PG option, so that the name is cleaned when you switch from PG to PF, is the expected behavior.

Reply:
do it